ROADMAP.md

```diff
@@ -0,0 +1,38 @@
+# Roadmap for 24.10 from April 2024 until October 2024
```

Comment:
should we have a checklist here to cross off all the things that we have completed?

Reply:
Interesting idea. I would maybe keep the tracking in JIRA, not to have another competing source of truth?

Reply:
I was more thinking about external presentation since external collaborators won't have access to Jira. But looking at it, the headings is clear enough on what's in the past, so probably not worth doing it